eng/tools/dependency-testing/README.md

```diff
@@ -71,3 +71,20 @@ rush rebuild
 ```
 
 Note : If the above step fails, you can reset the repo: `git clean -f -x -d` (Warning: this will delete all unversioned files including those ignored by gitignore. Backup any .env files and push any commits you wanted to etc)
+
```

Comment:
NIT

Can it say “Troubleshooting guide” or something?

And since this is not the complete debugging guide, can you put some subsection like “Case 1:  This is not a module error… follow below to mitigate the error”

Reply:
done